src/languageHandler.js

```diff
@@ -64,75 +61,112 @@ export function _t(...args) {
 }
 
 /*
- * Translates stringified JSX into translated JSX. E.g
- *    _tJsx(
- *        "click <a href=''>here</a> now",
- *        /<a href=''>(.*?)<\/a>/,
- *        (sub) => { return <a href=''>{ sub }</a>; }
- *    );
+ * Translates text and optionally also replaces XML-ish elements in the text with e.g. React components
+ * @param {string} text The untranslated text, e.g "click <a>here</a> now to %(foo)s".
+ * @param {object} variables Variable substitutions, e.g { foo: 'bar' }
+ * @param {object} tags Tag substitutions e.g. { 'a': (sub) => <a>{sub}</a> }
  *
- * @param {string} jsxText The untranslated stringified JSX e.g "click <a href=''>here</a> now".
- * This will be translated by passing the string through to _t(...)
+ * The values to substitute with can be either simple strings, or functions that return the value to use in
```

Comment:
Looking at the code, looks like they could probably be react components too? So you could just pass them in rather than functions that return them if it's a node that doesn't contain text? In fact, presumably you could even do these by passing it into `vars` rather than `tags`?